configure.ac

```diff
@@ -38,7 +38,7 @@ LIBBLKID_VERSION_INFO=`expr $LIBBLKID_LT_MAJOR + $LIBBLKID_LT_MINOR`:$LIBBLKID_L
 dnl libuuid version
 LIBUUID_VERSION="$PACKAGE_VERSION_MAJOR.$PACKAGE_VERSION_MINOR.$PACKAGE_VERSION_RELEASE"
 LIBUUID_LT_MAJOR=1
-LIBUUID_LT_MINOR=3
+LIBUUID_LT_MINOR=4
```

Comment:
I think we do not need to change the version; we use symbol versioning, and it should be enough.

Reply:
Because of symbol versioning you don't need to modify the soname (Major version number). But if you add new symbols, you should at least bump the minor version number.
But I can revert that if you want.

Reply:
We never do that ;-) The soname is unchanged for years.